example-get-started/generate.sh

```diff
@@ -19,146 +19,118 @@ fi
 mkdir -p $REPO_PATH
 pushd $REPO_PATH
 
-# (https://dvc.org/doc/install)
-
 virtualenv -p python3 .venv
 export VIRTUAL_ENV_DISABLE_PROMPT=true
 source .venv/bin/activate
 echo '.venv/' > .gitignore
 
-pip install "dvc[s3]"
-
-# https://dvc.org/doc/tutorials/get-started#initialize
+pip install gitpython
+pip install "git+https://github.com/iterative/dvc#egg=dvc[all]"
 
 git init
+cp $HERE/code/README.md .
 git add .
 git commit -m  "Initialize Git repository"
-
 git tag -a "0-git-init" -m "Git initialized."
 
+
 dvc init
 git commit -m "Initialize DVC project"
-
 git tag -a "1-dvc-init" -m "DVC initialized."
 
-# https://dvc.org/doc/tutorials/get-started/data-versioning
 
 mkdir data
 dvc get https://github.com/iterative/dataset-registry \
         get-started/data.xml -o data/data.xml
-
-# https://dvc.org/doc/tutorials/get-started/data-versioning#start-tracking-data
-
 dvc add data/data.xml
 git add data/.gitignore data/data.xml.dvc
 git commit -m "Add raw data"
-
 git tag -a "2-track-data" -m "Data file added."
 
-# https://dvc.org/doc/tutorials/get-started/data-versioning#configure-remote-storage
 
-# Remote active on this env only, for writing to HTTP redirect below.
 dvc remote add -d --local storage s3://dvc-public/remote/get-started
-
-# Actual remote for generated project (read-only). Redirect of S3 bucket above.
 dvc remote add -d storage https://remote.dvc.org/get-started
```

Comment:
You didn't find these comments useful?

Reply:
I think I removed it along other comments (that were not very useful since we have tags + hard to maintain up to date since they had links in them). Don't have a strong opinion about this one.

Reply:
I liked them. Helped me understand why there were 2 remotes in this project. I presonaly don't need then any more but in general I think they're useful comments.

Reply:
Yep, np! I'll get these two back.